mysql/datadog_checks/mysql/databases_data.py

```diff
@@ -169,8 +167,7 @@ def _cursor_run(self, cursor, query, params=None):
     @tracked_method(agent_check_getter=agent_check_getter)
     def _fetch_database_data(self, cursor, start_time, db_name):
         tables = self._get_tables(db_name, cursor)
-        tables_chunks = list(get_list_chunks(tables, self.TABLES_CHUNK_SIZE))
-        for tables_chunk in tables_chunks:
+        for tables_chunk in get_list_chunks(tables, self.TABLES_CHUNK_SIZE):
```

Comment:
`get_list_chunks` is a generator function that yields chunks of `TABLES_CHUNK_SIZE` lists. This is a tiny refactor that will avoid immediately generating a new list of chunk lists and instead generate the chunks from the tables list on each loop